libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -118,7 +118,7 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @Param: SQUAWK
     // @DisplayName: Squawk code
     // @Description: VFR squawk (Mode 3/A) code is a pre-programmed default code when the pilot is flying VFR and not in contact with ATC. In the USA, the VFR squawk code is octal 1200 (hex 0x280, decimal 640) and in most parts of Europe the VFR squawk code is octal 7000. If an invalid octal number is set then it will be reset to 1200.
-    // @Unit: octal
+    // @Units: octal
```

Comment:
This fix should be squashed into the previous commit instead of being inside a *Tools* commit.